src/main/kotlin/com/hypto/iam/server/service/CredentialService.kt

```diff
@@ -21,14 +21,15 @@ import org.koin.core.component.inject
 
 class CredentialServiceImpl : KoinComponent, CredentialService {
     private val repo: CredentialsRepo by inject()
+    private val usersService: UsersService by inject()
     private val idGenerator: ApplicationIdUtil.Generator by inject()
 
     override suspend fun createCredential(
         organizationId: String,
         userId: String,
         validUntil: String?
     ): Credential {
-        val userHrn = ResourceHrn(organizationId, "", IamResources.USER, userId)
+        val userHrn = ResourceHrn(usersService.getUser(organizationId, userId).hrn)
```

Comment:
Can we add a test cases for this please.

Reply:
Our cognito mock returns successfully for every getUser requests, so any tests for this will return success

Reply:
Can't the global mocks added in `AbstractContainerBaseTest` be overridden within a test case?